server/graphql/v2/collection/AccountCollection.js

```diff
@@ -13,6 +18,29 @@ const AccountCollection = new GraphQLObjectType({
       nodes: {
         type: new GraphQLList(Account),
       },
+      stats: {
```

Comment:
Add a description and make clear that stats are computed on the returned results, not everything.